### DISCUSSION TITLE
VLC HTTP Port Forwarding - User/Password Prompt?

### DISCUSSION THREAD

**Discussion by mrhhats** [GerryDazoo/Slinger#169] [Q&A] [answered]
Hi there! did the required port forwarding. Running on a windows OS and outside the network I get a username/password prompt in VLC when i use the http://url:8080/Slingbox

Am I missing something? I don't see a user/pass defined in a config or a default one 

<img width="565" alt="image" src="https://user-images.githubusercontent.com/111146190/197822659-574d9941-5bce-4b0c-a8d1-84339d27a9dc.png">

**Comment by GerryDazoo** [comment]
Yes. You URL is all wrong  http://url:8080/Slingbox
Should be http://IP_ADDRESS_OF_SERVER:8080/slingbox     No capital "S"

**Reply by JoeM2016** [reply]
It might be a VLC setting. I struggled with the same issue while using VLC on a different app, sorry I do not recall how I elimanated it.  Try just using admin as pw. The setting might be burried in the tools tab, preferences, interface, privacy / network all radio settings. Maybe try reset preferences.  Recently I installed VLC on a new pc and had no pw issues.

**Comment by mrhhats** [comment, marked as answer]
apologies for the wasted post :( turns out i missed another firewall rule with port 8080 that was directing traffic to the wrong internal ip.

**Reply by mrhhats** [reply]
Also, many thanks for doing this for the community!!
